src/Microsoft.DotNet.Tools.Compiler/Program.cs

```diff
@@ -39,7 +39,8 @@ public static int Main(string[] args)
             // Native Args
             var native = app.Option("-n|--native", "Compiles source to native machine code.", CommandOptionType.NoValue);
             var arch = app.Option("-a|--arch <ARCH>", "The architecture for which to compile. x64 only currently supported.", CommandOptionType.SingleValue);
-            var ilcArgs = app.Option("--ilcargs <ARGS>", "String to pass directory to ilc in native compilation.", CommandOptionType.SingleValue);
+            var ilcArgs = app.Option("--ilcargs <ARGS>", "String to pass directly to ilc in native compilation.", CommandOptionType.SingleValue);
+            var ilcPath = app.Option("--ilcpath <PATH>", "Directory with custom built ilc native compilation package.", CommandOptionType.SingleValue);
```

Comment:
I would rephrase this to "Path to the folder containing custom built ILCompiler."